tests/templates/kuttl/logging/kafka-vector-aggregator-values.yaml.j2

```diff
@@ -0,0 +1,55 @@
+---
+role: Aggregator
+service:
+  ports:
+  - name: api
+    port: 8686
+    protocol: TCP
+    targetPort: 8686
+  - name: vector
+    port: 6123
+    protocol: TCP
+    targetPort: 6000
+customConfig:
+  api:
+    address: 0.0.0.0:8686
+    enabled: true
+  sources:
+    vector:
+      address: 0.0.0.0:6000
+      type: vector
+      version: "2"
+  transforms:
```

Comment:
`automaticLogConfigBrokerPrepare` and `customLogConfigBrokerPrepare` are missing.

Reply:
good point thanks! https://github.com/stackabletech/kafka-operator/pull/547/commits/9ca65f25e6380342f3a02478882a9ab41def2ee6